androidApp/src/main/java/dev/johnoreilly/confetti/sessiondetails/SessionDetailsView.kt

```diff
@@ -113,13 +115,15 @@ fun SessionDetailView(
                     Spacer(modifier = Modifier.size(16.dp))
 
                     Text(
+                        modifier = Modifier.padding(horizontal = horizontalPadding),
                         text = session.startsAt.toTimeString(session.endsAt),
                         color = MaterialTheme.colorScheme.onSurface,
                         style = MaterialTheme.typography.labelLarge
                     )
 
                     session.room?.name?.let { roomName ->
                         Text(
+                            modifier = Modifier.padding(horizontal = horizontalPadding),
```

Comment:
Duplicating these does not feel quite correct, but the issue here is that I need to set the click modifier before the padding so that the padding does not limit the ripple. So I have to move the padding out of the column. Is there a better way to do this? 🤔